templates/zerver/for-open-source.html

```diff
@@ -243,17 +243,17 @@ <h1>
                             Zulip is <a href="https://github.com/zulip">100%
                             open-source software</a>, with no "open core"
                             catch. We work hard to make it <a href="https://zulip.readthedocs.io/en/latest/production/install.html">easy to set up</a>,
-                            <a href="https://zulip.readthedocs.io/en/stable/production/export-and-import.html#backups">backup
-                            </a>, and <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html">maintain
-                                      </a> a self-hosted Zulip installation, where you
+                            <a href="https://zulip.readthedocs.io/en/stable/production/export-and-import.html#backups">backup </a>,
+                            and <a href="https://zulip.readthedocs.io/en/stable/production/upgrade-or-modify.html">maintain </a>
```

Comment:
It's probably a bug that there's a trailing space inside the `<a>` tag,  which probably appears in rendering; can you tweak that?

Reply:
I fixed those.  All the for-open/for-company/for-* templates could probably use a quick human pass.  I think there are places where a pretty-printer could more aggressively clean up the templates, but there are complications like Twitter tweets, where I don't know if a formatter would get obnoxious.  There are also strange idioms that look slightly intentional in nature, but I'm not entirely sure they're needed.